src/web_interface/filter.py

```diff
@@ -256,9 +237,9 @@ def data_to_chart_with_value_percentage_pairs(data, limit=10):  # pylint: disabl
     return result
 
 
-def set_color_list_for_data_to_chart(color_list, value_list):
+def set_color_list_for_data_to_chart(color_list, value_list, limit: int):
```

Comment:
Hm. Not sure I like this being here (since it's only use is in `jinja_filter.py`. But if we view this module as a de facto helper module it's fine I guess.

Reply:
The split between `filter.py` and `jinja_filter.py` seems rather arbitrary to me anyway. Maybe we find better ways to split them up.

Reply:
Anyway ... moved to jinja_filter.py